src/Controls/src/Core/Platform/Windows/CollectionView/ItemContentControl.cs

```diff
@@ -284,6 +284,10 @@ protected override WSize MeasureOverride(WSize availableSize)
 		{
 			if (_renderer == null)
 			{
+				// Make sure we supply a real number for height otherwise virtualization won't function
+				if (double.IsFinite(availableSize.Width) && !double.IsFinite(availableSize.Height))
```

Comment:
Just want to make sure this is ok in CV scenarios where we have horizontal list scrolling set as the layout.